src/main.rs

```diff
@@ -505,6 +505,7 @@ fn run() -> Result<(), failure::Error> {
             "{} Retrieve your user info and test your auth config",
             emoji::SLEUTH
         )))
+        .subcommand(SubCommand::with_name("tail").about("Aggregate logs from production worker"))
```

Comment:
will need to add `.arg(env)` here I believe

Reply:
i think it already works...

Reply:
```console
$ wrangler tail --env dev
error: Found argument '--env' which wasn't expected, or isn't valid in this context

USAGE:
    wrangler tail

For more information try --help
```

it might be handled in main.rs but you also have to add the argument explicitly to the long clap chain here

Reply:
https://github.com/cloudflare/wrangler/issues/1179